pKa_challenge_instructions.md

```diff
@@ -0,0 +1,115 @@
+
+## Due Date
+
+Your predictions must be uploaded on the D3R SAMPL6 web-page by January 10, 2018. The experimental results will be released immediately after the challenge closes. You must use the provided templates to upload your predictions to the [SAMPL website](https://drugdesigndata.org/about/sampl6). Additional information on using these templates is provided below.
+
+## Computational prediction methods
+
+You may use any method(s) you like to generate your predictions; e.g., molecular mechanics or quantum mechanics based methods, QSPR, empirical pKa prediction tools etc.
+
+## 24 small molecules included in the pKa challenge
+![pKa_challenge_small_molecules](images/pKa_challenge_small_molecules.jpg)
+A list of SAMPL6 pKa challenge small molecules canonical isomeric SMILES and molecule IDs can be found here: [/physical_properties/pKa/molecule_ID_and_SMILES.csv](physical_properties/pKa/molecule_ID_and_SMILES.csv). Counterions, where present in solid formulations, were included in canonical isomeric SMILES for the sake of completeness, although no significal effect is expected from the presence of chloride counterions as experiments were conducted in ionic-strength adjusted medium with KCl.
+
+## Experimental details
+pKa measurements were collected using spectrophotometric pKa measurements with Sirius T3 instrument by Mehtap Isik from the Chodera Lab at MSKCC with the support of Merck Rahway, Preformulation Department. 
+
+Small molecules were purchased in powder form. 10 mg/ml DMSO solutions were prepared and used as stock solutions for preparation of samples, where 1-5 uL of 10 mg/ml DMSO stock solution is diluted in 1.5 mL ionic-strength adjusted water (0.15 M KCl). pH titrations with acid (0.5 M HCl, 0.15 M KCl) and base (0.5 M KOH, 0.15 M KCl) and cosolvent addition (80% Methanol, 0.15 M KCl) were performed in an automated fashion with Sirius T3.
+
+[UV-metric pKa measurement protocol of Sirius T3](http://www.sirius-analytical.com/science/pka) evaluates pKas based on change of absorbance in the UV region of the spectrum while a pH titration is being performed between pH 1.8 and 12.2. UV-absorbance spectra of small molecule chromophores can change as a function of pH,  based on the protonation state change of groups close to the chromophore. Change in multiple wavelengths as a function of pH is fitted to a model to determine pKas. Number of pKas in the model is selected based on better fit between experimental and modeled microstates curves. 
+
+This method is capable of measuring pKas between 2 and 12, when protonatable groups are at a maximum distance of 4-5 atoms away from chromophores, so that a change of protonation state affects the absorbance of UV-chromophore. We have selected compounds where titratable groups are close to potential chromophores (conjugated rings), but it is possible to completely miss detection of a pKa value if a titratable group is not in interaction with a UV-chromophore.  
```

Comment:
Maybe clarify "it is possible to completely miss detection of a pKa value if a titratable group is not in interaction with a UV-chromophore." I think what you mean is something more like, "it is possible that our experimental results will not detect the titration of certain cites (and will thereby neglect the pKa of those sites) if a titratable group is not near enough to a UV chromophore."

Reply:
Done.